kb/2023-07-06/package_your_own_toolbox_image.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Package Your Own Toolbox Image
+description: How to package your own toolbox
+slug: package_your_own_toolbox_image
+authors:
+  - name: Vicente Cheng
+    title: Senior Software Engineer
+    url: https://github.com/Vicente-Cheng
+    image_url: https://github.com/Vicente-Cheng.png
+tags: [debug, harvester, container]
+hide_table_of_contents: false
+---
+
+Harvester OS is the immutable OS. That means you cannot install any packages on it. Although we provide the way to install packages (https://docs.harvesterhci.io/dev/troubleshooting/os#how-can-i-install-packages-why-are-some-paths-read-only). But it is not recommended to install packages on Harvester OS.
```

Comment:
```suggestion
Harvester OS is an immutable OS. That means you cannot install any packages on it. Although we provide a way to install packages (https://docs.harvesterhci.io/dev/troubleshooting/os#how-can-i-install-packages-why-are-some-paths-read-only), it is not recommended to install packages on Harvester OS.
```
It would be nice to provide the reason why it's not recommended.

Reply:
Added more description for it